tests/test_Jetscape.py

```diff
@@ -379,10 +379,6 @@ def test_Jetscape_print(jetscape_file_path, output_path):
     jetscape.print_particle_lists_to_file(output_path)
     assert filecmp.cmp(jetscape_file_path, output_path)
     os.remove(output_path)
-
-def test_Jetscape_get_sigmaGen(jetscape_file_path):
-    jetscape = Jetscape(jetscape_file_path)
-    assert jetscape.get_sigmaGen() == (0.000314633,6.06164e-07)
     
```

Comment:
Why is the test `test_Jetscape_get_sigmaGen` removed?

Reply:
Probably a mistake. I will add it again.